src/components/UserSelector.tsx

```diff
@@ -42,6 +46,14 @@ export const UserSelector: React.FC<Props> = ({
   // when the Dopdown is closed
   }, [expanded]);
 
+  useEffect(() => {
+    // Load users when the component mounts
+    // Assuming getUsers is a function that returns a Promise
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)